src/System.Linq/src/System/Linq/Enumerable.cs

```diff
@@ -2865,6 +2978,23 @@ public bool Contains(TKey key)
                 } while (g != _lastGrouping);
             }
         }
+        
+        public IGrouping<TKey, TElement>[] ToArray()
```

Comment:
Good catch indeed. I noticed public in couple places, but those were in internal classes, so I figured it was ok...